plugins/woocommerce/tests/php/src/Blocks/StoreApi/Routes/AdditionalFields.php

```diff
@@ -1891,7 +1891,7 @@ public function test_place_order_required_address_field() {
 		$data     = $response->get_data();
 
 		$this->assertEquals( 400, $response->get_status(), print_r( $data, true ) );
-		$this->assertEquals( \sprintf( 'There was a problem with the provided shipping address: %s is required', $label ), $data['message'], print_r( $data, true ) );
+		$this->assertEquals( \sprintf( 'There was a problem with the provided shipping address: %s is required', $label ), $data['data']['params']['shipping_address'], print_r( $data, true ) );
```

Comment:
cc @opr

Reply:
I think this is fine, as long as the error message gets to the front end ok